docs/UserGuide.md

```diff
@@ -259,17 +274,17 @@ WARNING: This will override the pre-existing data with the new data you entered.
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [a/STUDENT_NUMBER] [e/EMAIL] [m/MODULE] [g/GROUP] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Exit** | `exit`
-**Find** | `find PREFIX/KEYWORD [MORE_KEYWORDS] [PREFIX/KEYWORD [MORE_KEYWORDS]]`<br> e.g., `find n/James Jake a/A0217`
-**List** | `list`
-**Mail** | `mail`<br> e.g., `mail e/johndoe@example.com`
-**Mail All** | `mailall`
-**Help** | `help`
-**New Task** | `newtask DESCRIPTION by/DEADLINE` <br> e.g., `newtask Do homework by/2022-03-21T12:34`
-**Delete Task** | `deltask INDEX` <br> e.g., `deltask 3`
+
+| Action         | Format, Examples                                                                                                                                                |
+|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**        | `add n/NAME p/PHONE_NUMBER e/EMAIL m/MODULE g/GROUP [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com m/CS2100 g/W12 t/friend t/colleague` |
+| **Clear**      | `clear`                                                                                                                                                         |
+| **Delete**     | `delete INDEX`<br> e.g., `delete 3`                                                                                                                             |
+| **Edit**       | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [m/MODULE] [g/GROUP] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                            |
+| **Exit**       | `exit`                                                                                                                                                          |
+| **Find**       | `find PREFIX/KEYWORD [MORE_KEYWORDS] [PREFIX/KEYWORD [MORE_KEYWORDS]]`<br> e.g., `find n/James Jake`                                                            |
+| **List**       | `list`                                                                                                                                                          |
+| **Mail Index** | `mail-index`<br> e.g., `mail 2`                                                                                                                                 |
+| **Mail X**     | `mail-x`<br> e.g., `mail e/johndoe@example.com n/Alex`                                                                                                          |
+| **Mail All**   | `mail-all`                                                                                                                                                      |
+| **Help**       | `help`                                                                                                                                                          |
```

Comment:
Could you help to include the newTask and delTask back? Thank you!

Reply:
Added back! Thank you for pointing out.